android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/home/ListAdapter.java

```diff
@@ -54,7 +55,15 @@ public void onBindViewHolder(@NonNull RecyclerViewHolder holder, int position) {
         holder.cashFlowValue.setTypeface(Typeface.defaultFromStyle(Typeface.BOLD));
         Format formatter = new SimpleDateFormat("dd.MM.yyyy", Locale.GERMANY);
         holder.date.setText(formatter.format(cashflow.getTimestamp()));
+        holder.materialCard.setOnClickListener(view -> {
```

Comment:
Could be possible but we have never talked about this possibility.
My idea was to show the manual adding layout with the filled in data that @ChrisEwert is currently implementing in the other branch. In that way we might be able to avoid implementing a details view / expanding system by its own.
Anyways, it is not decided what we are going to do with the onclick event/ how the details view will look like. 
I added the Toast because this is what we had implemented earlier.